Formula/i/iccmax.rb

```diff
@@ -0,0 +1,154 @@
+class Iccmax < Formula
+  desc "Reference implementation tools for iccMAX"
+  homepage "https://github.com/InternationalColorConsortium/DemoIccMAX"
+  url "https://github.com/InternationalColorConsortium/DemoIccMAX/archive/eb63a92c7428c6a8fc0ca522cb779a7e72186ef8.tar.gz"
+  version "2.1.26"
```

Comment:
This still needs to be an actual tag. We need the version to be stable, and the best way to do that is tagged versions.